pkg/proxy/iptables/proxier_test.go

```diff
@@ -374,6 +374,8 @@ func TestDeleteEndpointConnectionsIPv6(t *testing.T) {
 
 const testHostname = "test-hostname"
```

Comment:
hm, what do you mean?

One place I was inconsistent is whether the test cases used constant strings

```
		output:   "10.180.0.1:80",
```

or generated them from local variables:

```
		output:   fmt.Sprintf("%s:%d", epIP, svcPort),
```

Some of the test functions had local variables to use and some didn't, so I just followed the lead of the existing code in each function. But I never decided which approach I liked better...

Reply:
> E.g. could we do something like:

we already do; this was fixed in https://github.com/kubernetes/kubernetes/pull/108251/commits/d74df127e9cfc3d437311151a96a36bedcc176f2 as part of #108251 (inspired by [your comment here](https://github.com/kubernetes/kubernetes/pull/106497#discussion_r772690791). (And it's documented in the lines immediately above the line you commented on.)

They aren't quite perfect because I was (originally) trying to avoid illegally using "real" IPs, but there were only so many easily-differentiated private/reserved IP ranges to pick from. So I made node IPs be 192.168.0.x and ExternalIPs be 192.168.99.x (which kind of makes sense because in many configurations, ExternalIPs are on the same subnet as the nodes). And "cluster-external and allowed" and "cluster-external but blocked" only differ in 1 bit, but that doesn't really matter because we only make that distinction in a few places and the test names always make it clear what's going on...

But then I ran out of distinguishable ranges and ended up using 1.2.3.4, 5.6.7.8, and 9.10.11.12 for load balancer IPs, violating my "no real IPs" rule. So I guess we could use other real IP ranges to better differentiate the other ranges.

Or we could just switch to IPv6 and encode the meanings in leetspeek (`70ad:ba7a:ce00::1`)...

Reply:
I hate that I love `10ad:ba1a:ce00::1`, but let's not :)